sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/MetricDataMapper.java

```diff
@@ -153,7 +153,12 @@ public static void updateMetricPointBuilder(
                 throw new IllegalArgumentException("metric data type '" + type + "' is not supported yet");
         }
 
-        pointBuilder.setValue(pointDataValue * 1000); // upstream changes millis to seconds and we need to send millis back to breeze
+        // new http semconv metrics use seconds, but we want to send milliseconds to Breeze
+        if (metricData.getName().equals("http.server.request.duration") || metricData.getName().equals("http.client.request.duration")) {
```

Comment:
I think I'd add `&& isPreAggregatedStandardMetric` just to be safe in case we decide to export these as regular metrics (in addition to standard preagg metrics)